src/fileStore.js

```diff
@@ -114,30 +114,30 @@ module.exports = class FileStore {
         if (!block || !block.blockNumber) { throw new Error(`Bad block: ${JSON.stringify(block)}`) }
         const path = this.getBlockPath(block.blockNumber)
         const raw = JSON.stringify(block)
-        this.log(`Saving block ${block.blockNumber} to ${path}: ${this.sanitize(raw)}`)
+        this.log(`Saving block ${block.blockNumber} to ${path}`)
         if (await fs.exists(path)) {
             this.log(`Overwriting block ${block.blockNumber}!`)
         }
-        return fs.writeFile(path, raw,(err) => {
+        return fs.writeFile(path, raw, (err) => {
```

Comment:
actually this might be wrong now I look at it, not sure if mz supports mixed callback/promise code, it should probably be:

```js
await fs.writeFile(path, raw)
return this.makeLatestSymlink(block.blockNumber)
```